lib/rules/at-rule-semicolon-space-before/README.md

```diff
@@ -0,0 +1,45 @@
+# at-rule-semicolon-space-before
+
+Require a single space or disallow whitespace before the semicolons of regular (non-nested) at-rules.
```

Comment:
I think I'm failing lingo. I was trying to talk about at-rules like this one:

```css
@font-face {
  font-family: \"MyFont\";
  src: url(\"myfont.woff2\") format(\"woff2\");
}
```

(I got the terminology from [CSS Tricks](https://css-tricks.com/the-at-rules-of-css/))

These at rules are ignored because of the call to `hasBlock`, as is the case for at-rule-semicolon-newline-after.

Should I amend this to say "at-rules without declaration blocks" or some such?